orc8r/cloud/go/obsidian/swagger/remote_spec.go

```diff
@@ -24,17 +24,20 @@ import (
 	"github.com/golang/glog"
 )
 
-type remoteSpec struct {
-	// service name of the remoteSpec
+// RemoteSpec identifies a remote spec
+type RemoteSpec struct {
```

Comment:
@hcgatewood You mentioned in a previous comment that the struct should be made public but since we only ever initialize the object with NewRemoteSpec() we don't necessarily need to have the object be public right? We can still access the objects methods without it the object itself being exported.